crates/vapp/src/state.rs

```diff
@@ -61,18 +61,6 @@ pub struct VAppState<A: Storage<Address, Account>, R: Storage<RequestId, bool>>
     ///
     /// Fees earned by the protocol are sent to this address.
     pub treasury: Address,
```

Comment:
I think for consistency, we should also put `treasury` in the RequestProofRequestBody then nuke this